bower.json

```diff
@@ -1,8 +1 @@
-{
-  "name": "bespin-ui",
-  "dependencies": {
-    "ember": "~2.9.0",
-    "ember-cli-shims": "0.1.3",
-    "bootstrap": "~3.3.5"
-  }
-}
+{"name":"bespin-ui","dependencies":{}}
```

Comment:
Good catch! Looks like Ember no longer uses bower by default. They've been phasing it out over the last few releases. Version 2.13 doesn't create bower.json for new projects. Since we have no other dependencies on bower, I'm removing the `bower.json` file and the installation step to install it.

You can also remove the `bower_components` directory from your local copy.